common/throttle.h

```diff
@@ -12,6 +12,7 @@ class throttle {
     uint64_t m_time_window;
     uint64_t m_time_slice;
     uint64_t m_slice_num;
+    bool m_no_limit;
```

Comment:
SInce limit is uint64_t unsigned int, using -1UL(UINT64_MAX) to present un-throttled is a designed solution, while `limit = 0` means cut-off control.